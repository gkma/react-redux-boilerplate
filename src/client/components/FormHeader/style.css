.form-header {
  background-color: violet;
  border-radius: 1rem 1rem 0 0;
  margin: auto;
  width: 65%;
}

.form-header-text {
  color: #ffffff;
  font-family: "Franklin Gothic Medium", "Arial Narrow", Arial, sans-serif;
  margin: 0;
  padding: 2em;
  text-align: center;
}

.form-header-text > * {
  margin: 0;
}
