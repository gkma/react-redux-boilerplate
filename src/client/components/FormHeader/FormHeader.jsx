import React from 'react';

import Heading from '../../shared/Heading';

import './style.css';

const FormHeader = ({ pageStatus }) => {
  return (
    <header className='form-header g-form'>
      <div className='form-header-text'>
        <Heading
          headline='A-List ðŸŒŽ Tours'
          subHeadline={`Please fill out form below (${pageStatus}/3)`}
        />
      </div>
    </header>
  );
};

export default FormHeader;
