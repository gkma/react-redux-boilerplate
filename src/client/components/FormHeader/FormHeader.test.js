import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import FormHeader from ".";

Enzyme.configure({ adapter: new Adapter() });

describe("<FormHeader />", () => {
  const wrapper = shallow(<FormHeader />);

  it("renders component successfully", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("component name is 'header'", () => {
    expect(wrapper.name()).toEqual("header");
  });
});
