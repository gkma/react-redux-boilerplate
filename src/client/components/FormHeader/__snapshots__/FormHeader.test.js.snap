// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormHeader /> renders component successfully 1`] = `ShallowWrapper {}`;
